common/ferrostar/src/models.rs

```diff
@@ -171,8 +171,17 @@ pub struct CourseOverGround {
 }
 
 impl CourseOverGround {
-    pub fn new(degrees: u16, accuracy: Option<u16>) -> Self {
-        Self { degrees, accuracy }
+    /// # Arguments
+    ///
+    /// - degrees: The direction in which the user's device is traveling, measured in clockwise degrees from
+    ///            true north (N = 0, E = 90, S = 180, W = 270).
+    /// - accuracy: the accuracy of the course value, measured in degrees.
+    pub fn new(degrees: f64, accuracy: Option<u16>) -> Self {
```

Comment:
This is a larger change than necessary, but it cleaned things up a bit. 

A couple things:

1. Are you comfortable trusting input to be 0...360 now that `geo::Bearing` does so?
2. It's maybe a little weird that this `new` takes in an f64 while storing it as u16, but it does reflect how we're using the API in practice.

I can make changes if you'd prefer.

Reply:
Ooh, thanks for asking!

1. Yes. [iOS](https://developer.apple.com/documentation/corelocation/cllocationdirection) and [Android](https://developer.android.com/reference/android/location/Location#getBearing()) also make guarantees about the range of valid values (iOS uses negative to indicate an error/unavailability, and we already handle that [here](https://github.com/stadiamaps/ferrostar/blob/065a7d2ab94ae1227d593a254ea5896cc4315be1/apple/Sources/FerrostarCore/Extensions/CoreLocationExtensions.swift#L16). I DO think we should document this though just so it's clear to users. The debug_assert is also a good call. After a think over it, I think this is right since it's going to be pretty hard to get a negative value on most platforms.
2. Yeah, agreed. It probably makes sense to accept a float and then do conversion in just one place so that we know it's consistent.

Reply:
EDIT: Looks like I have to file a bug in Android for contradictory docs. The bearing setter does mod 360 there too but the getter docs are contradictory on whether the 360 is inclusive or exclusive haha.